src/ML-POD/compute_podfit.h

```diff
@@ -0,0 +1,214 @@
+/***************************************************************************
+               CESMIX-MIT Project
+
+ Contributing authors: Ngoc-Cuong Nguyen (cuongng@mit.edu, exapde@gmail.com)
+ ***************************************************************************/
+
+#ifdef COMPUTE_CLASS
+// clang-format off
+ComputeStyle(podfit,CPODFIT);
+// clang-format on
+#else
+
+#ifndef LMP_COMPUTE_PODFIT_H
+#define LMP_COMPUTE_PODFIT_H
+
+#include "compute.h"
+#include "pod.h"
+
+namespace LAMMPS_NS {
+
+class CPODFIT : public Compute {
+private:
+
+  class NeighList *list;
+  std::vector<std::string> globVector(const std::string& pattern, std::vector<std::string> & files);
+
+  bool is_a_number(std::string line);
+
+public:
+  struct datastruct {
```

Comment:
This seems to be almost the same struct as used by the pair style. Why not use one common definition?

Reply:
This is declared in `fitpod_command.h` now, and it is only used for that command style.